README.md

```diff
@@ -99,14 +99,19 @@ Then you can test your app either with the Speculos emulator :
 
 ```bash
 # Run your app on Speculos
-root@656be163fe84:/app# speculos build/nanos/bin/app.elf --model nanos
+bash$ speculos build/nanos/bin/app.elf --model nanos
 ```
 
 Or you can run your Ragger functional tests if you have implemented them :
 
 ```bash
+# Creating a virtualenv so that the non-root user can install Python dependencies
+bash$ python -m virtualenv venv --system-site-package
+bash$ source ./venv/bin/activate
+# Install tests dependencies
+(venv) bash$ pip install -r tests/requirements.txt
 # Run ragger functional tests
-root@656be163fe84:/app# pytest tests/ --tb=short -v --device nanos --display
+(venv) bash$ python -m pytest tests/ --tb=short -v --device nanos --display
```

Comment:
We can also call `pytest` directly (using `python -m pytest ...` is not necessary)